### PR TITLE
Peer testing eric

### LINKED ISSUE #449: Run peer testing session and fill in parts labelled "Everyone" on report
This task involves running the two sessions, adding the video link to the report, and filling any part of the report labelled with "Everyone". 

Deadline: March 1

### PR DESCRIPTION
**Pull request by wrc9817**

### CODE REVIEW

**Review by aleksssk — APPROVED** [review]
Nice work!

File: documentation/peer-testing-2/peer-testing-report.md [hunk 1]
```diff
@@ -31,9 +31,13 @@ In total, **\_** individuals took part in our peer testing sessions. The members
 **To be completed after the peer testing sessions**
 
 #### High
-
+- Server sometimes accidentally shut down for unexpected reason.
+- Need an extra tab for mentors to view the history of their declined mentee.
```
Comment (katmartel) [inline documentation/peer-testing-2/peer-testing-report.md]: That's an interesting point. The client hasn't mentioned the ability to view declined mentees, so I don't  think it's an important aspect for the system, so maybe we could put that as a low priority, which could be a bonus feature for once we finalize the higher priority functionalities.

File: documentation/peer-testing-2/peer-testing-report.md [hunk 2]
```diff
@@ -31,9 +31,13 @@ In total, **\_** individuals took part in our peer testing sessions. The members
...
+- Mentor user should not to logout and login to update their suggested mentees
```
Comment (katmartel) [inline documentation/peer-testing-2/peer-testing-report.md]: Minor typo: I think 'have' should be in between 'not' and 'to' 
```suggestion
- Mentor user should not have to logout and login to update their suggested mentees
```

File: documentation/peer-testing-2/peer-testing-report.md [hunk 3]
```diff
@@ -42,4 +46,11 @@ In total, **\_** individuals took part in our peer testing sessions. The members
 
 ### Testimonials from the Participants to Explain the Graph
 
+- After finishing filling AdminForm and verfying new account, user was confused for what to do next because the page remained on 'verification email has been sent'. 
+- User tried to fill the sample forms of mentor and mentee as an admin, which should not happen.
+- "what about google.com" user said when he noticed that he needed to enter an URL in avaliability field, which indicates that using tooltip to hint users is not enought and specific.
```
Comment (katmartel) [inline documentation/peer-testing-2/peer-testing-report.md]: Minor typos
```suggestion
- "what about google.com" a user said when he noticed that he needed to enter a URL in the availability field, which indicates that using a tooltip to hint to users is not specific enough.
```

**Review by katmartel — APPROVED** [review]
I've provided a few minor suggestions. Nice work :)

File: documentation/peer-testing-2/peer-testing-report.md (hunk 1 above)
Comment (wrc9817) [reply documentation/peer-testing-2/peer-testing-report.md]: Agree! Thank you for reviewing!

**Review by jpabadir — APPROVED** [review]

**Pull request merged**